src/edictor/app/js/glosses.js

```diff
@@ -117,10 +105,10 @@ GLOSSES.modify_entry = function(event, node, type, idx, jdx) {
     this.unmodify_entry(node, type);
     return;
   }
-  else if (event.keyCode != 13 && event != 'click' && [37, 38, 39, 40].indexOf(event.keyCode) == -1)  {
+  else if (eevent != 'click' && [13, 37, 38, 39, 40].indexOf(event.keyCode) == -1)  {
```

Comment:
```
eevent
```

Is this a typo or is this deliberately done? Did you check what happens if the event is `click`?